bolt/test/X86/reorder-uncond-jmp.s

```diff
@@ -0,0 +1,38 @@
+# REQUIRES: system-linux
+
+# RUN: llvm-mc -filetype=obj -triple x86_64-unknown-unknown \
+# RUN:   %s -o %t.o
+# RUN: link_fdata %s %t.o %t.fdata
+# RUN: link_fdata %s %t.o %t.fdata2 "FDATA2"
+# RUN: %host_cc %cflags %t.o -o %t.exe -Wl,-q
+# RUN: llvm-bolt %t.exe -data %t.fdata -reorder-blocks=cache+ -print-finalized \
+# RUN:    -uncond-jmp-reorder -o %t.out | FileCheck %s
```

Comment:
Even when you set the pass to run by default, leave the flag here, so it's explicit that we are testing this pass.